cmds/rm/rm_test.go

```diff
@@ -93,3 +93,29 @@ func Test_rm_2(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+// using f flag
+func Test_rm_3(t *testing.T) {
```

Comment:
Go uses camel case, and let's give it a descriptive name: TestForceRemove? (I realize the rest of the file is wrong, too... we can fix that later, or you can fix it now if you want.)

Reply:
Yeah, I don't like this either. But the rest of the tests in the same package are written in this manner and thus decided to follow the 'When in Rome, do as the Romans do' proverb for the sake of consistency.

Also, #327 is supposed to fix this, but fair enough. I will name this one properly.

Reply:
Just to clarify. I intend to rewrite these tests using sub-tests (i.e. feel free to assign #327 to me), because currently the tests pass thanks to the order they execute of and the repetitive setting up in each tests. Didn't do it in this PR in order to stay in scope.